index.css

```diff
@@ -0,0 +1,608 @@
+.grid-divs {
+  grid-row-gap: 3%;
+}
+
+.grid-divs img {
+  width: 100%;
+  cursor: pointer;
+}
+
+.grid-divs div {
+  margin-bottom: 5%;
+}
+
+.grid-divs a:hover {
+  color: #f42;
+}
+
+.second-p {
+  padding: 2% 0;
+}
+
+/* Section 3 div design */
+
+.majordiv {
+  position: relative;
+}
+
+.majordiv h1 {
+  text-align: center;
+  background-color: #f42;
+  color: white;
+  padding: 1%;
+}
+
+.div1 {
+  background-color: #ffebe8;
+  text-indent: 40%;
+  padding: 3%;
+}
+
+.div2 p {
+  padding: 2%;
+}
+
+.div1 p:nth-child(1) {
+  font-family: Arial, Helvetica, sans-serif;
+  font-weight: 900;
+  font-size: 20px;
+}
+
+.div2 {
+  background-color: #fffaf8;
+  padding: 0 5% 10% 5%;
+}
+
+.img img {
+  position: absolute;
+  width: 20%;
+  height: 15%;
+  top: 3%;
+  left: 5%;
+}
+
+.lastp {
+  padding-bottom: 5%;
+}
+
+/* section4 styles */
+
+.section4 {
+  margin-top: 2%;
+  margin-bottom: 2%;
+}
+
+.grid-background {
+  height: 20rem;
+  background-image: url(./images/apple.png);
+  background-position: center;
+  background-size: cover;
+  flex-direction: column;
+  justify-content: flex-end;
+  padding: 2%;
+  color: white;
+}
+
+.grid-flex {
+  background-color: #ffebe8;
+  padding-left: 2%;
+  margin-top: 1%;
+  justify-content: space-between;
+}
+
+.grid-flex img {
+  width: 100%;
+  height: 100%;
+}
+
+.flex-img {
+  width: 50%;
+}
+
+.section4 li {
+  border-bottom: 1px solid lightgray;
+  padding: 2% 0 2%;
+  list-style: none;
+}
+
+.section4 li::before {
+  content: "\2022";
+  color: #f42;
+  font-weight: bolder;
+}
+
+/* Section5 style */
+
+.section5 {
+  margin: 7% auto;
+}
+
+.second-p .strikethrough {
+  text-decoration: line-through;
+  font-size: 15px;
+}
+
+/* Footer Styles */
+
+.ftr1 {
+  flex-direction: column;
+  justify-content: center;
+  text-align: center;
+  background-color: #131313;
+  height: 170px;
+}
+
+.links {
+  margin-bottom: 2%;
+}
+
+.ftr2 {
+  color: white;
+  background-color: black;
+  padding: 4%;
+}
+
+.ftr2 div {
+  background-color: #146ff8;
+  float: right;
+  padding: 2%;
+  border-radius: 50%;
+  margin-top: -4%;
+}
+
+/* media queries */
+
+/* tablets */
+
+@media screen and (max-width: 1024px) {
+  .right-nav img,
+  .right-nav button {
+    display: none;
+  }
+
+  .alignment {
+    width: 90%;
+    margin: 0 auto;
+  }
+
+  .title {
+    margin-top: 5%;
+  }
+
+  .block {
+    display: block;
+  }
+
+  #div5 {
+    margin-bottom: 2%;
+  }
+
+  #TNW {
+    display: flex;
+    width: 100%;
+  }
+
+  .majordiv h1,
+  .div1 p {
+    font-size: 20px;
+  }
+
+  #TNW p {
+    margin-left: 30%;
+  }
+
+  .smallimg a {
+    font-size: 20px;
+  }
+
+  .majordiv .grey {
+    font-size: 12px;
+  }
+
+  .div2 .lastp {
+    font-size: 15px;
+    text-align: left;
+  }
+
+  ul li {
+    font-size: 10px;
+  }
+
+  .div2 a {
+    color: black;
+  }
+}
+
+/* phones */
+
+@media screen and (max-width: 600px) {
+  main {
+    margin: 0;
+    padding: 0;
+  }
+
+  body {
+    overflow-x: hidden;
+    width: 100%;
```

Comment:
- [ ] As the requirements in pathwright state, you should use the mobile-first approach, that means that the default is the small screen and no media queries are used for this size, next you should state the media queries for tablet @media screen and (min-width: 768px) and lastly the media queries for desktop @media screen and (min-width: 1024px). [You did desktop-first and established max-widths for your media queries]. Please correct this.